torchvision/utils.py

```diff
@@ -40,10 +40,6 @@ def make_grid(
 
     Returns:
         grid (Tensor): the tensor containing grid of images.
-
-    Example:
```

Comment:
Can we link to the galeries from the docs? I think it would be a nice thing to have

Reply:
We can!
We can add manual links to an example and even reference a specific plot of a given example.

But in this case, I made sure that every torchvision function and class that is used in an example will automatically generate a "minigallery"

https://github.com/pytorch/vision/blob/d6fee5a4c4518ddb11f3dc1b52c102a7bdad5b49/docs/source/conf.py#L284-L289

![image](https://user-images.githubusercontent.com/1190450/117663256-5db1a280-b198-11eb-8ff1-4edfc95e3d2f.png)

Reply:
Awesome!